apps/outlook-rooms-addin/src/app/components/not-found/not-found.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="flex flex-1 flex-col z-0 bg-gray-50 h-full overflow-y-auto">
+    <section
+        class="flex flex-col border-b border-gray-300 py-5 h-full items-center justify-center"
+    >
+        <div class="mx-3 flex flex-row">
+            <img
+                src="assets/not-found.svg"
+                alt="graphic of magnifying glass"
+                width="200px"
+                class="items-center"
+            />
+        </div>
+        <div class="mx-3 flex flex-row">
+            <div class="mb-3">
+                <span class="text-3xl font-bold mr-auto text-gray-700">
+                    Page not found</span
+                >
+            </div>
+        </div>
+        <div class="mx-3 flex flex-row">
+            <span class="flex flex-row">
+                The page
+                <span class="font-bold mx-1"> {{ router.url }} </span> was not
+                found.
+            </span>
+        </div>
+    </section>
+</div>
```

Comment:
This component is simple enough that it might be better for the template to be in the `.ts` file